diff -Nur shotwell-0.11.0.orig/plugins/Makefile.plugin.mk shotwell-0.11.0/plugins/Makefile.plugin.mk
--- shotwell-0.11.0.orig/plugins/Makefile.plugin.mk	2011-08-23 19:19:18.000000000 +0100
+++ shotwell-0.11.0/plugins/Makefile.plugin.mk	2011-09-04 21:31:10.000000000 +0100
@@ -32,8 +32,8 @@
 CFILES := $(notdir $(SRC_FILES:.vala=.c))
 OFILES := $(notdir $(SRC_FILES:.vala=.o))
 
-CFLAGS := `pkg-config --print-errors --cflags $(EXT_PKGS)` -nostdlib -export-dynamic $(PLUGIN_CFLAGS)
-LDFLAGS := `pkg-config --print-errors --libs $(EXT_PKGS)` $(LDFLAGS)
+LOCAL_CFLAGS := `pkg-config --print-errors --cflags $(EXT_PKGS)` -nostdlib -export-dynamic $(PLUGIN_CFLAGS)
+LOCAL_LDFLAGS := `pkg-config --print-errors --libs $(EXT_PKGS)` $(LDFLAGS)
 DEFINES := -D_VERSION='"$(PLUGINS_VERSION)"' -DGETTEXT_PACKAGE='"shotwell"'
 
 all: $(PLUGIN).so
@@ -51,10 +51,10 @@
 	@
 
 $(OFILES): %.o: %.c $(CFILES)
-	$(CC) -c $(CFLAGS) $(DEFINES) -I../.. $(CFILES)
+	$(CC) -c $(LOCAL_CFLAGS) $(DEFINES) -I../.. $(CFILES)
 
 $(PLUGIN).so: $(OFILES)
-	$(CC) $(CFLAGS) -Wl,--no-as-needed $(LDFLAGS) $(OFILES) -I../.. -shared -o $@
+	$(CC) $(LOCAL_CFLAGS) -Wl,--no-as-needed $(LOCAL_LDFLAGS) $(OFILES) -I../.. -shared -o $@
 
 .PHONY: cleantemps
 cleantemps:
