diff -dur -x '*~' shotwell-0.13.1.orig/Makefile shotwell-0.13.1/Makefile
--- shotwell-0.13.1.orig/Makefile	2012-10-08 11:15:07.000000000 +0200
+++ shotwell-0.13.1/Makefile	2012-10-08 11:17:13.000000000 +0200
@@ -558,8 +558,8 @@
 	mkdir -p $(DESTDIR)$(PREFIX)/bin
 	$(INSTALL_PROGRAM) $(PROGRAM) $(DESTDIR)$(PREFIX)/bin
 	$(INSTALL_PROGRAM) $(THUMBNAILER_BIN) $(DESTDIR)$(PREFIX)/bin
-	mkdir -p $(DESTDIR)$(PREFIX)/libexec/shotwell
-	$(INSTALL_PROGRAM) $(MIGRATOR_BIN) $(DESTDIR)$(PREFIX)/libexec/shotwell
+	mkdir -p $(DESTDIR)$(PREFIX)/${LIB}/shotwell
+	$(INSTALL_PROGRAM) $(MIGRATOR_BIN) $(DESTDIR)$(PREFIX)/${LIB}/shotwell
 	mkdir -p $(DESTDIR)$(PREFIX)/share/shotwell/icons
 	$(INSTALL_DATA) icons/* $(DESTDIR)$(PREFIX)/share/shotwell/icons
 	mkdir -p $(DESTDIR)$(PREFIX)/share/icons/hicolor/scalable/apps
diff -dur -x '*~' shotwell-0.13.1.orig/src/AppDirs.vala shotwell-0.13.1/src/AppDirs.vala
--- shotwell-0.13.1.orig/src/AppDirs.vala	2012-10-05 19:56:07.000000000 +0200
+++ shotwell-0.13.1/src/AppDirs.vala	2012-10-08 11:18:23.000000000 +0200
@@ -109,7 +109,7 @@
                 // not installed yet - use wherever we were run from
                 libexec_dir = get_exec_dir();
             } else {
-                libexec_dir = File.new_for_path(Resources.PREFIX + "/libexec/shotwell");
+                libexec_dir = File.new_for_path(Resources.PREFIX + "/" + Resources.LIB + "/libexec/shotwell");
             }
         }
 
