--- shotwell-0.26.1/configure.ac.gitver	2017-04-15 15:26:08.372007826 +0200
+++ shotwell-0.26.1/configure.ac	2017-04-15 15:26:19.075341151 +0200
@@ -13,7 +13,7 @@ m4_define([api_version],[major_version])
 
 AX_IS_RELEASE([git-directory])
 
-m4_define([git_version],[m4_esyscmd(git rev-parse HEAD 2>/dev/null)])
+m4_define([git_version],[])
 
 dnl ***********************************************************************
 dnl Initialize autoconf
