--- shotwell-0.9.1/plugins/Makefile.plugin.mk.orig	2011-04-04 19:54:27.000000000 +0200
+++ shotwell-0.9.1/plugins/Makefile.plugin.mk	2011-04-05 08:52:34.000000000 +0200
@@ -32,8 +32,8 @@
 CFILES := $(notdir $(SRC_FILES:.vala=.c))
 OFILES := $(notdir $(SRC_FILES:.vala=.o))
 
-CFLAGS := `pkg-config --print-errors --cflags $(EXT_PKGS)` -nostdlib -export-dynamic $(PLUGIN_CFLAGS)
-LDFLAGS := `pkg-config --print-errors --libs $(EXT_PKGS)` $(LDFLAGS)
+LOCAL_CFLAGS := `pkg-config --print-errors --cflags $(EXT_PKGS)` -nostdlib -export-dynamic $(PLUGIN_CFLAGS)
+LOCAL_LDFLAGS := `pkg-config --print-errors --libs $(EXT_PKGS)` $(LDFLAGS)
 DEFINES := -D_VERSION='"$(PLUGINS_VERSION)"' -DGETTEXT_PACKAGE='"shotwell"'
 
 all: $(PLUGIN).so
@@ -51,10 +51,10 @@
 	@
 
 $(OFILES): %.o: %.c $(CFILES)
-	$(CC) -c $(CFLAGS) $(DEFINES) -I../.. $(CFILES)
+	$(CC) -c $(LOCAL_CFLAGS) $(DEFINES) -I../.. $(CFILES)
 
 $(PLUGIN).so: $(OFILES)
-	$(CC) $(CFLAGS) $(LDFLAGS) $(OFILES) -I../.. -shared -o $@
+	$(CC) $(LOCAL_CFLAGS) $(LOCAL_LDFLAGS) $(OFILES) -I../.. -shared -o $@
 
 .PHONY: cleantemps
 cleantemps:
