--- shotwell-0.13.1.orig/src/Dialogs.vala	2012-10-05 19:58:13.000000000 +0200
+++ shotwell-0.13.1/src/Dialogs.vala	2012-10-08 12:20:20.000000000 +0200
@@ -32,8 +32,8 @@
 public bool confirm_warn_developer_changed(int number) {
     Gtk.MessageDialog dialog = new Gtk.MessageDialog.with_markup(AppWindow.get_instance(),
         Gtk.DialogFlags.MODAL, Gtk.MessageType.WARNING, Gtk.ButtonsType.NONE,
-        "<span weight=\"bold\" size=\"larger\">%s</span>".printf(ngettext("Switching developers will undo all changes you have made to this photo in Shotwell",
-        "Switching developers will undo all changes you have made to these photos in Shotwell", number)));
+        "<span weight=\"bold\" size=\"larger\">%s</span>", ngettext("Switching developers will undo all changes you have made to this photo in Shotwell",
+        "Switching developers will undo all changes you have made to these photos in Shotwell", number));
 
     dialog.add_buttons(Gtk.Stock.CANCEL, Gtk.ResponseType.CANCEL);
     dialog.add_buttons(_("_Switch Developer"), Gtk.ResponseType.YES);
