--- shotwell-0.32.14/src/meson.build.orig	2025-11-14 15:05:11.000000000 +0100
+++ shotwell-0.32.14/src/meson.build	2025-11-18 21:53:29.821872153 +0100
@@ -39,10 +39,14 @@ face_sources = (['faces/FacesBranch.vala
                      'faces/FacesTool.vala'])
 
 shotwell_deps = [gio, gee, sqlite, gtk, sqlite, posix, gphoto2,
-                 gstreamer_pbu, gudev, gexiv2, gmodule, unity,
+                 gstreamer_pbu, gudev, gexiv2, gmodule,
                  libraw, libexif, sw_plugin, webpdemux, webp, version,
                  portal, math]
 
+if unity.found() and get_option('unity_support')
+  shotwell_deps += [unity]
+endif
+
 subdir('metadata')
 subdir('publishing')
 subdir('video-support')
